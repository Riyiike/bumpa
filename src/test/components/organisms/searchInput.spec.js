import React from "react";
import { shallow } from 'enzyme'
import SearchInput from "../../../components/organisms/searchInput";

jest.mock('react-router-dom', () => ({
    ...jest.requireActual('react-router-dom'),
    useSearchParams: () => [],

}))

jest.mock('react', () => ({
    ...jest.requireActual('react'),
    useContext: () => ({
    }),
    useState: () => [],

}))


describe('test mainPage', () => {
    it('test mainPage', () => {
        const btn = shallow(<SearchInput />)
        expect(btn).toMatchSnapshot();
    })
})