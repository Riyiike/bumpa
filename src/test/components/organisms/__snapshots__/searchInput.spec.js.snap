// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test mainPage test mainPage 1`] = `
<div
  className="flex items-center gap-3 p-3 px-5 w-full max-w-xs rounded-lg shadow-md bg-White dark:bg-DarkBlue text-DarkGray dark:text-VeryLightGray mb-3 sm:mb-0"
>
  <svg
    fill="currentColor"
    height="16"
    viewBox="0 0 16 16"
    width="16"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"
    />
  </svg>
  <input
    className="focus:outline-none bg-[transparent] grow dark:placeholder:text-VeryLightGray"
    onChange={[Function]}
    placeholder="Search for a country..."
    type="text"
  />
</div>
`;
